tests/element_test_vectors.rs

```diff
@@ -419,83 +422,90 @@ mod impl_display_for_element_tests {
     }
 }
 
+const ELEMENT_GLOBAL_SKIP_LIST: SkipList = &[
```

Comment:
🗺️ It appears that these are identical across all `ElementApi` implementations--I'd bet this was a vestige from when we had Ion C lurking.